src/amo/components/AddonMoreInfo/index.js

```diff
@@ -91,8 +91,25 @@ export class AddonMoreInfoBase extends React.Component<Props> {
       );
     }
 
-    const currentVersion = addon.current_version;
     const lastUpdated = addon.last_updated;
+    const currentVersion = addon.current_version;
+    const { license } = currentVersion;
+    let licenseHtml = null;
```

Comment:
You could probably name this variable `versionLicenseLink`, no?

Reply:
well it's not always a link, right? so I thought this was more fitting...

Reply:
`versionLicenseLink` is the key used a few line below.

Reply:
Altho, I see that's what we are calling it in the function..

Reply:
okay I see..I'll just rename this